Lib/test/test_exceptions.py

```diff
@@ -319,15 +319,16 @@ def baz():
         check('(yield i) = 2', 1, 2)
         check('def f(*):\n  pass', 1, 7)
 
-    @unittest.skipIf(ctypes.sizeof(ctypes.c_ssize_t) > ctypes.sizeof(ctypes.c_int),
-                     "Py_ssize_t is bigger than INT_MAX, so this is unreachable")
+    @unittest.skipIf(ctypes.sizeof(ctypes.c_int) >= ctypes.sizeof(ctypes.c_ssize_t),
+                     "INT_MAX is bigger than Py_ssize_t, so this is unreachable")
     @support.requires_resource('cpu')
     @support.bigmemtest(support._2G, memuse=1.5)
     def testMemoryErrorBigSource(self):
```

Comment:
This probably needs another merge, since a `_size` argument has been added. Also, it might be good to request the size of the buffer we want and then use that. Something like:

```python3
@support.bigmemtest(2 ** (ctypes.sizeof(ctypes.c_int) * 8), memuse=1)
    def testMemoryErrorBigSource(self, size):
        padding = '        ' * (size // 8)
```

Reply:
Done.